src/pages/steps/EventTypeAndThemeStep.tsx

```diff
@@ -289,32 +314,34 @@ const EventTypeAndThemeStep = ({
                       row-gap: ${parseSpacing(3.5)()};
                     `}
                   >
-                    {types.map(({ id, name }) => (
-                      <Button
-                        width="auto"
-                        display="inline-flex"
-                        key={id}
-                        variant={ButtonVariants.SECONDARY_TOGGLE}
-                        onClick={() => {
-                          field.onChange({
-                            ...field.value,
-                            type: { id, label: name[i18n.language] },
-                          });
-                          onChange({
-                            ...field.value,
-                            type: { id, label: name[i18n.language] },
-                          });
-                        }}
-                        css={`
-                          &.btn {
-                            padding: 0.3rem 0.7rem;
-                            box-shadow: ${getGlobalValue('boxShadow.heavy')};
-                          }
-                        `}
-                      >
-                        {name[i18n.language]}
-                      </Button>
-                    ))}
+                    {(isCultuurkuurEvent ? cultuurkuurTypes : types).map(
```

Comment:
```suggestion
                    {types.map(
```
What @Anahkiasen said ⏫ You can move the filtering logic into the memoized types variable.
Since the mapping logic here remains the same